config/routes.rb

```diff
@@ -111,7 +111,9 @@
   get '/search/toggle_beta_search', as: :toggle_beta_search, to: 'search#toggle_beta_search'
 
   scope constraints: ->(_req) { TradeTariffFrontend::ServiceChooser.uk? } do
-    get 'exchange_rates(/:year)', to: 'exchange_rates#index', as: 'exchange_rates'
+    # param 'type' = [average | monthly | spot], 'year' is a query param.
```

Comment:
I think we can kill this note?

Reply:
yeah, we will add it in the API doc soon.